blueocean-core-js/src/js/bundleStartup.js

```diff
@@ -14,7 +14,7 @@ export function execute(done, config) {
         classMetadataProvider: (type, cb) => {
             const fetch = require('./fetch').Fetch;
             const fetchOptions = {
-                method: 'POST',
+                method: 'GET',
```

Comment:
?

Reply:
Oh thats actually #1920 

But the api for getting a specific class is GET only

Reply:
should I revert this change over here and get it merged separately in #1920 or get it merged here and close the other PR? I was just annoyed of always seeing those 404s when developing this feature so that is why I added it here.

Reply:
Reverting is going to be a mess, we'll just deal with the conflict after 1920, or ignore 1920